components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/main/java/org/wso2/carbon/identity/api/resource/mgt/dao/impl/APIResourceManagementDAOImpl.java

```diff
@@ -445,6 +475,18 @@ public List<Scope> getScopesByTenantId(Integer tenantId, List<ExpressionNode> ex
             appendFilterQuery(expressionNodes, filterQueryBuilder, true);
             String query = SQLConstants.GET_SCOPES_BY_TENANT_ID + filterQueryBuilder.getFilterQuery() +
                     SQLConstants.GET_SCOPES_BY_TENANT_ID_TAIL;
+            try {
+                if (OrganizationManagementUtil.isOrganization(tenantId) && !isInternalFiltering(filterQueryBuilder)) {
```

Comment:
According to the logic in `isInternalFiltering` method, internal_org.. scopes also get considered , is that means internal_org.. scopes can't be searched ?